autoextract_poet/pages.py

```diff
@@ -0,0 +1,41 @@
+from dataclasses import dataclass
+
+from web_poet.pages import ItemPage
+
+from autoextract_poet.inputs import (
+    ProductResponseData,
+    ProductListResponseData,
+)
+
+
+@dataclass
+class ProductPage(ItemPage):
```

Comment:
I think that's fine to keep base classes for pages out of scope for this PR. This looks like a separate task.

Reply:
Done.